indico/modules/users/controllers.py

```diff
@@ -639,3 +639,17 @@ class RHUserSearchInfo(RHProtected):
     def _process(self):
         external_users_available = any(auth.supports_search for auth in multipass.identity_providers.itervalues())
         return jsonify(external_users_available=external_users_available)
+
+
+class RHUserBlock(RHUserBase):
+    def _process_PUT(self):
+        self.user.is_blocked = True
+        logger.info("User %s blocked %s", session.user, self.user)
```

Comment:
Sorry for being nitpicky but we usually use single quotes.

Reply:
No problem! I thought the linter was picking this up.

Reply:
Does flake8 have a plugin which can check for this (single quotes, unless it would require escaping inside)? But even if it does, I think it'd require quite a few changes in existing files for English strings where we defaulted to double quotes for a while even if there were no apostrophes inside.

Reply:
Something like this?
https://pypi.org/project/flake8-quotes/

Reply:
Oh nice, this looks great! I just gave it a try and with some configuration it works pretty well to avoid stuff like unnecessary escapes.

unfortunately the "wrong type of quotes" check isn't really suitable for us:

```
$ flake8 indico/ | grep Q000 | wc -l
1366
```

a bit better for multiline strings, but that's more because we don't have that many which aren't docstrings..

```
$ flake8 indico/ | grep Q001 | wc -l
138
```

and even better when ignoring it for `indico/core/signals/` since the multiline strings there are basically docstrings (and thus double quotes are more appropriate).

```
$ flake8 indico/ | grep Q001 | wc -l
34
```

But I don't think it really matters for multiline strings, so I'm inclined to disable Q000 and Q001 globally and just let it complain about unnecessary escapes and docstrings not using double-quotes.

Too bad there's no way to run this check just for newly added code...